test/functional/p2p_blocksonly.py

```diff
@@ -74,6 +73,10 @@ def run_test(self):
 
         self.log.info('Check that the tx from forcerelay first_peer is relayed to others (ie.second_peer)')
         with self.nodes[0].assert_debug_log(["received getdata"]):
+            # NB: in practice, first_peer would never send us transactions since we're a blocksonly node.
+            # But if first_peer runs a modified version of Bitcoin Core to bypass relay checks and forward
```

Comment:
nit: s/forward/forwards

Reply:
My thinking was that `forward` was going in pair with `bypass` (both infinitive), but I'm not a native speaker, I can definitely apply that.

Reply:
Yeah first_peer is running a modified version of Bitcoin Core to bypass relay checks. Once it has done that the first_peer forwards the transactions. But as I said I don't think we are fussy on grammar for comments on this project and I wouldn't want to put people like you off from adding comments in future! So feel free to leave it. We'll see if Gleb has more substantial suggestions.